docs/conf.py

```diff
@@ -53,7 +53,7 @@ def setup(sphinx):
 
 # General information about the project.
 project = 'Solidity'
-copyright = '2016-2019, Ethereum'
+copyright = '2016-2020, Ethereum'
```

Comment:
Are there any other places in the source to update?

Reply:
There are plenty in license preambles. Probably they need to be updated as well.